hack/test-selinux.sh

```diff
@@ -20,8 +20,14 @@ NAME="$1"
 ## so keeping the label as nfs_t fits right. Package container-selinux by
 ## default adds rules for nfs_t context which allows container workloads to work as well.
 ## https://github.com/lima-vm/lima/pull/1965
+##
+## With recent integration[https://github.com/lima-vm/lima/pull/2474] with systemd-binfmt,
```

Comment:
"recent" does not make sense as it is unclear to readers when this was written

Reply:
I had attached the PR as well. Would that work?

Reply:
No, as people won't open all the links when they read the comment

Reply:
Understood. So what do you think would suit here well, I will update according to that.

Reply:
Just remove "recent"

Reply:
Done